Bottom: 5443b430419fc07b750ee98358ff646fef9e996a
Top:    434d1f79bbb4588b345db38c99d911a6ede96936
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-12-16 15:29:34 +0800

XDP: introduce XDP_HOLD


---

diff --git a/include/uapi/linux/bpf.h b/include/uapi/linux/bpf.h
index 7d9b283..4e177aa 100644
--- a/include/uapi/linux/bpf.h
+++ b/include/uapi/linux/bpf.h
@@ -530,6 +530,7 @@ enum xdp_action {
 	XDP_DROP,
 	XDP_PASS,
 	XDP_TX,
+	XDP_HOLD,
 };
 
 /* user accessible metadata for XDP packet hook
